### DISCUSSION TITLE
Daily chart layout

### DISCUSSION THREAD

**Discussion by Sp4rkR4t** [mcguirepr89/BirdNET-Pi#465] [Ideas]
I was just wondering if there was any reasoning why the daily chart section only shows the top 10 and bottom 10 in the chart, I've discovered my wife loves a data graph but she never looks at that section any more because it's not a complete at a glance picture.

I would propose a complete chart of the daily detection's and also that the common names be clickable to take you to that days recordings of those detection's, I feel this would give easier access to the information the system provides.

**Comment by Svardsten53** [comment]
I totally agree with you and when I mentioned this before someone was kind enough to make me a version that shows all the species. It is the species on the lower part of the list that are the most interesting, I think.
I would think it will be difficult to make the species clickable as the solution now is with a chart in the form of an image.

**Reply by Sp4rkR4t** [reply]
Ah yeah didn't notice the names were part of the image.

**Reply by ehpersonal38** [reply]
Do you have a link to that version? I could implement something in the config that lets you adjust the amount of species it displays.

**Comment by Svardsten53** [comment]
Please see issue [#219](https://github.com/mcguirepr89/BirdNET-Pi/issues/219). I have done some translations, but it looks like this on my [installation](https://birdnet.svardsten.se/)

**Reply by Sp4rkR4t** [reply]
Thanks, that's given me the pointers I needed to do it myself.

**Comment by CaiusX** [comment]
Hi @Sp4rkR4t 

If you go to Species Stats & click Single Day View you will get the following screen;

![image](https://user-images.githubusercontent.com/25373276/180222244-6e23784f-4ec7-4bb6-ac46-7b9898175208.png)

Clicking on Raw will give the same as the main page, and number of birds to show can be extended to the entire day.  You can also get previous days selecting Date to View.


If it gets too crowded, click on the double arrow at the top right of the figure to get a full page, which can be downloaded/saved.
![image](https://user-images.githubusercontent.com/25373276/180222989-785c5c25-8470-42b2-a9a4-2b67ef5538a4.png)

![image](https://user-images.githubusercontent.com/25373276/180223555-4f4b2c26-7339-4367-8001-a545df175581.png)
